File: FizzBuzz.cpp
#include <iostream>
using namespace std;

int main() {
	// Do FizzBuzz
	for (int i = 1; i <= 100; i++) {
		if (i % 15 == 0) {cout << "FizzBuzz";}
		else if (i % 3 == 0) {cout << "Fizz";}
		else if (i % 5 == 0) {cout << "Buzz";}
		else {cout << i;}
		cout << endl;
	}
	
	// Exit
	return 0;
}

File: FizzBuzz.lua
-- Do FizzBuzz
i = 1
while (i <= 100) do
	if (math.fmod(i,15) == 0) then
		print("FizzBuzz")
	elseif (math.fmod(i,3) == 0) then
		print("Fizz")
	elseif (math.fmod(i,5) == 0) then
		print("Buzz")
	else
		print(i)
	end
	i = i + 1
end